src/main/msc/emfat_file.c

```diff
@@ -231,6 +231,7 @@ static const char icon_file[] =
 #define CMA_TIME EMFAT_ENCODE_CMA_TIME(1,1,2018, 13,0,0)
 #define CMA { CMA_TIME, CMA_TIME, CMA_TIME }
 
+#if defined (USE_EMFAT_AUTORUN) || defined (USE_EMFAT_ICON) || defined (USE_EMFAT_README)
```

Comment:
Are you sure this function (`memory_read_proc`) is not needed if we only disable the icon and the autorun? Just from reading the name it seems like it is an important function that handles a part of the FAT32 file reads. If it only used to read the data of the 3 defines (`USE_EMFAT_AUTORUN`, `USE_EMFAT_ICON`, `USE_EMFAT_README`) then this is fine.

Reply:
I am sure. There is no other usage in the code base and without ifdefs linker complains that the function was unused.

Reply:
Yes, seems that function `memory_read_proc` is used only in this file